src/form-builder/RadioGroup.tsx

```diff
@@ -20,6 +20,12 @@ export function RadioGroup(props: RadioGroupProps): JSX.Element {
     withValidation as FieldHookConfig<string>
   );
 
+  const onChangeHandler = (e: Event) => {
```

Comment:
Any reason we use `onChangeHandler` here instead of `onChange` like the other components?